<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>462</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>441</width>
      <height>261</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox_temp">
       <property name="title">
        <string>temp</string>
       </property>
       <widget class="QPushButton" name="pushButton_temp">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>获取温度</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_temp">
        <property name="geometry">
         <rect>
          <x>272</x>
          <y>20</y>
          <width>151</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2_power">
       <property name="title">
        <string>power</string>
       </property>
       <widget class="QPushButton" name="pushButton_2_power">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>功率</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_2_power">
        <property name="geometry">
         <rect>
          <x>272</x>
          <y>20</y>
          <width>151</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3_bias_read">
       <property name="title">
        <string>read bias</string>
       </property>
       <widget class="QPushButton" name="pushButton_3_bias_read">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>读取</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_3_bias_read">
        <property name="geometry">
         <rect>
          <x>272</x>
          <y>20</y>
          <width>151</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_4_bias_write">
       <property name="title">
        <string>write bias</string>
       </property>
       <widget class="QPushButton" name="pushButton_4_bias_write">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>设置</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_4_bias_write">
        <property name="geometry">
         <rect>
          <x>272</x>
          <y>20</y>
          <width>151</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>2</width>
      <height>2</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2"/>
   </widget>
   <widget class="QPushButton" name="pushButton_translation">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>平移</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_translation">
    <property name="geometry">
     <rect>
      <x>282</x>
      <y>310</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>462</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
